src/seismometer/controls/decorators.py

```diff
@@ -2,13 +2,79 @@
 """
 As for all of the utils subpackage, no other custom subpackages should be referenced.
 """
-import builtins
+import hashlib
+import logging
+import os
+import shutil
 from functools import wraps
+from inspect import signature
+from pathlib import Path
 from typing import Any, Callable
 
-from IPython.display import display
+from IPython.display import HTML
 from ipywidgets import Widget
 
+logger = logging.getLogger("seismometer")
+SEISMOMETER_CACHE_DIR = Path("./.seismometer_cache")
```

Comment:
Nit:  Are there cases when the leading ./ is required?
![image](https://github.com/epic-open-source/seismometer/assets/36572844/c916b275-9050-49d6-a550-b1748c611432)

Reply:
FINC